keps/sig-scheduling/5027-dra-admin-controlled-device-attributes/README.md

```diff
@@ -0,0 +1,772 @@
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+<!--
+Pick one more of these and delete the rest.
+-->
+
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+<!--
+Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+implementation difficulties, etc.).
+-->
+
+### Dependencies
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### Does this feature depend on any specific services running in the cluster?
+
+<!--
+Think about both cluster-level services (e.g. metrics-server) as well
+as node-level agents (e.g. specific version of CRI). Focus on external or
+optional services that are needed. For example, if this feature depends on
+a cloud provider API, or upon an external software-defined storage or network
+control plane.
+
+For each of these, fill in the following—thinking about running existing user workloads
+and creating new ones, as well as about cluster-level services (e.g. DNS):
+  - [Dependency name]
+    - Usage description:
+      - Impact of its outage on the feature:
+      - Impact of its degraded performance or high-error rates on the feature:
+-->
+
+### Scalability
+
+Applying patches to devices scales with `number of ResourceSlicePatches` *
+`number of devices` when CEL selectors need to be evaluated. Without them,
+filtering scales with `number of ResourceSlicePatches` * `number of
+ResourceSlices` but then may still need to compare device names and of course
+modify selected devices.
+
+###### Will enabling / using this feature result in any new API calls?
+
+A fixed, small number of clients (primarily the scheduler) need to start
+watching ResourceSlicePatches.
+
+###### Will enabling / using this feature result in introducing new API types?
+
+ResourceSlicePatches must be created explicitly by admins or controller
+operated by admins. Kubernetes itself does not create them.
+
+The number of ResourceSlicePatches is expected to be orders of
+magnitude smaller than the number of ResourceSlices.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No.
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+No.
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+Pod scheduling should be as fast as would be without this feature, because in
+both cases it starts with listing all devices. That information is local can
+comes either from an informer cache or a cache of patched devices.
```

Comment:
Should this read as something like this?
```suggestion
both cases it starts with listing all devices. That information is local and
comes either from an informer cache or a cache of patched devices.
```

Reply:
A bit weird when reading it if miss "and"

Reply:
Fixed.